lte/gateway/c/core/oai/include/mme_app_ue_context.h

```diff
@@ -41,10 +41,16 @@
 #include "lte/gateway/c/core/oai/include/s6a_messages_types.h"
 #include "lte/gateway/c/core/oai/include/sgw_ie_defs.h"
 #include "lte/gateway/c/core/oai/lib/bstr/bstrlib.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
 #include "lte/gateway/c/core/oai/lib/hashtable/hashtable.h"
 #include "lte/gateway/c/core/oai/lib/hashtable/obj_hashtable.h"
+#include "lte/gateway/c/core/oai/tasks/mme_app/mme_app_sgs_fsm.hpp"
```

Comment:
mme_app_ue_context.h is included in many other .hpp/.cpp files which is causing undefined reference to many functions in mme_app_sgs_fsm.hpp. Instead of tracing all the files in which mme_app_ue_context.h is included and then wrapping it under extern C, I have added mme_app_sgs_fsm.hpp under extern C

Reply:
Could you add comment with mentioning github issue id i.e #13096